tests/fixtures.py

```diff
@@ -8,6 +8,7 @@
 
 MODELS_PATH = "https://raw.githubusercontent.com/DARPA-ASKEM/simulation-integration/main/data/models/"
 DATA_PATH = "https://raw.githubusercontent.com/DARPA-ASKEM/simulation-integration/main/data/datasets/"
+DATA_PATH2 = "https://raw.githubusercontent.com/ciemss/pyciemss/sa-load-data-tests/docs/source/sa-testing-notebooks/"
```

Comment:
This path does not look stable, especially once the branch is deleted. Could you please push these data files to the existing `DATA_PATH` above.

Reply:
@SamWitty I just made a PR to add the dataset to the original `MODELS_PATH`